modules/libcom/RTEMS/epicsNtp.c

```diff
@@ -0,0 +1,86 @@
+/*************************************************************************\
+* (C) 2014 David Lettier.
+* http://www.lettier.com/
+* EPICS BASE is distributed subject to a Software License Agreement found
+* in file LICENSE that is included with this distribution.
```

Comment:
This file and the associated header appear to be heavily-modified versions of the C source from [this repo](https://github.com/lettier/ntpclient) which is provided under the BSD 3-clause license. Incorporating this code into EPICS is thus legal, but we can't change the license terms of someone else's software and the wording of our standard header might imply that we are trying to do that. Please remove lines 4 and 5 from both files, and add this instead:
```
* SPDX-License-Identifier: BSD-3-Clause
```